src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -1058,7 +1065,7 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
                 CheckTypeArgumentsCore(typeArguments, operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, workingSet);
             }
 
-            static void CheckTypeArgumentsCore(ImmutableArray<ITypeSymbol> typeArguments, IOperation operation, OperationAnalysisContext context,
```

Comment:
We should either retain this as static and pass the new bool parameter to the required methods OR remove all the extra parameters that are shared between these local functions and containing method (`AnalyzeOperation`)

Reply:
Good point, removing all extra parameters from local functions